src/components/Navbar/Navbar.tsx

```diff
@@ -1,13 +1,59 @@
-import React,{useState} from "react";
+import React, { useState } from "react";
+import { searchValueReducer } from "../../redux/reducer/jobs";
+import { DispatchType } from "../../redux/store/store";
+import { useDispatch } from "react-redux";
 
 const Navbar = () => {
-  const [filter] = useState({
+  const [filter, setFilter] = useState({
     jobLevel: "default",
     salary: "default",
     workingModel: "default",
     companyType: "default",
-  })
+  });
+  const dispatch: DispatchType = useDispatch();
+  const handleChangeFilter = (
+    e: React.ChangeEvent<HTMLSelectElement>,
+    type: string
+  ): void => {
+    if (type === "jobLevel") {
+      setFilter({ ...filter, jobLevel: e.target.value });
+    }
+    if (type === "salary") {
+      setFilter({ ...filter, salary: e.target.value });
+    }
+    if (type === "workingModel") {
+      setFilter({ ...filter, workingModel: e.target.value });
+    }
+    if (type === "companyType") {
+      setFilter({ ...filter, companyType: e.target.value });
+    }
```

Comment:
This repeat code.
Should be using name in select.
const { name, value } = e.target.value;
setFilter({ ...filter, [name]: value });